### ISSUE TITLE
@windowed macro for recurrence networks

### ISSUE THREAD

**Issue by pucicu** [JuliaDynamics/RecurrenceAnalysis.jl#114]
In the previous version, I could calculate the network measure transitivity using `rqa`, allowing me to use the `@windowed`macro. Now, transitivity is moved to `rna`, but it seems that the `@windowed`macro cannot be applied to `rna`. Would be nice if this would also be applicable or consistent to `rqa`.

**Comment by heliosdrm** [comment]
Technically this might be done, but I wonder: `rna` looks the signal as a complex network of points, without time structure; so, does it make sense to do a windowed analysis?

**Comment by pucicu** [comment]
Yes it makes sense. You can consider evolving networks which also change over time.
